src/main/java/com/training/sample/pages/Assignment2.java

```diff
@@ -0,0 +1,7 @@
+package com.training.sample.pages;
+
+public class Assignment2 {
+    public String stringReplace(String string1, String string2) {
```

Comment:
give meaningful Variable name ie newName,oldName,and use suffix get in method as it is returning string value